### DISCUSSION TITLE
Settings organization

### DISCUSSION THREAD

**Discussion by Donkey-Doug** [Docile-Alligator/Infinity-For-Reddit#1044] [Ideas]
There are many entries in the settings page:

![Screenshot_20211004-004253_Infinity](https://user-images.githubusercontent.com/50382414/135773895-bc8cc5ec-e5de-498f-ac69-61d28da2c8bd.png)

Some of these entries only have one option, like lazy mode interval. That is, there is an entire seperate section for one setting.

I suggest the following structure:

- appearance
    - interface
    - theme
- behavior
    - gestures & buttons
    - video
    - lazy mode interval
    - Sort types
- NSFW & spoiler
- notifications
- download location
- data saving
- miscellaneous
- advanced
- about

Also, please add a search field to browse the settings.

**Comment by Wladefant** [comment]
My proposal moved to #1080 

Yeah I agree that the settings need to be reorganized as they are very contraintuitive and have duplicates:

My proposal would be something like (not complete):

Settings:
Search bar 
![Screenshot_20220826-110402.jpg](https://user-images.githubusercontent.com/79611093/186868809-2886a4ba-27ac-4279-9356-30088f57d948.jpg)

- Network
  **Data Saving Mode**
  - Data Saving Mode (dropdown)
  - Disable Image Preview
  - Only Disable Preview in Video and Gif Posts

- Security
  - Require Authentication to Show Accounts
  - Secure Mode
  - App Lock
  - App Lock Timeout

- Storage 
  - Download location 
  - Import/Export
     **Database** (Title)
![Screenshot_20220826-132240_AntennaPod.jpg](https://user-images.githubusercontent.com/79611093/186893029-5723df2e-ffc1-4f34-969a-54e89288ee26.jpg)
    - Delete All Subreddits
    - Delete All Users
    - Delete All Sort Types 
    - Delete All Post Layouts
    - Delete All Themes
    - Delete All Front Page Scrolled Positions
    - Delete All Read Posts

     **Settings** (Title)
    - Delete All Legacy Settings
    - Reset All Settings
    - Backup Settings
    - Restore Settings

- Notifications
  - Enable Notifications (toggle)
  - Check Notifications Interval
  - Show Device Notification Settings

 ---
**Project** (Title)
- Documentation (Link to Github wiki)
- Forum (support, questions, feature requests) (Link to github discussions)
- Contribute (Link to contribution guidelines)
- Report Bug (Link to github issues)
- About
  - Infinity For Reddit version (subtitle: 2.6.1)
  - Source Code
  - Rate on Google Play
  - Subreddit 
  - Opensource libraries (instead of Acknowledgement)
  - Credits
  - View Changelog (or what's new)
  - Privacy Policy 
  - Reddit User Agreement

**Comment by Wladefant** [comment]
@Donkey-Doug Please add each feature request to its own post
